src/rebar_prv_shell.erl

```diff
@@ -331,8 +331,8 @@ reread_config(State) ->
     case find_config(State) of
         no_config ->
             ok;
-        ConfigList ->
-            _ = rebar_utils:reread_config(ConfigList),
+        Config ->
+            _ = rebar_utils:reread_config([Config]),
```

Comment:
No that's what I'm saying, but the API of `rebar_file_utils:consult_config` did change its return value. So I Was wondering how to fix this without needing to change `reread_config`. The answer seems to be to keep `rebar_file_utils:consult_config` the way you modified it, but wraps *that* result in a list *within consult_config* so that stuff does not break and does not require further modification anywhere.

I.e. we have to maintain the type signature for that function to still be `consult_config(rebar_state:t(), string()) -> [[tuple()]].` whereas right now you patched it to be `consult_config(rebar_state:t(), string()) -> [tuple()].` and that return value is being patched by hand at the call site.